server/test/dashboard.test.js

```diff
@@ -0,0 +1,58 @@
+import request from 'supertest';
+import { stub } from 'sinon';
+import server from '../app';
+import database from '../config/database';
+import Board from '../models/boardModel';
+import Dashboard from '../models/dashboardModel';
+
+describe('Dashboard screen APIs', () => {
+  describe('Add dashboard API', () => {
+    let requestBody;
+    afterEach(() => {
+      server.close();
+      database.disconnectDB();
+    });
+    beforeEach(() => {
+      requestBody = {
+        name: 'Pesto',
+      };
+    });
+    afterAll(() => {
+      server.close();
+      database.disconnectDB();
+    });
+
+    test('Should return 404 if user id not passed in params found', (done) => {
```

Comment:
404 means not found, but while creating a dashboard if a user is not passed, it should be 400

Reply:
After Sign in implementation, I will take user ID from session instead of passing. For now, if the user id is not passed in params, the route will not found and the server will give 404 error.

Reply:
in that case it should throw 403, unauthorized, not 404

Reply:
yes, But the sign-in functionality is in progress, will update test cases and API after sign-in implementation.

Reply:
Does this test case pass?